contents/handbook/why-does-posthog-exist.md

```diff
@@ -6,29 +6,63 @@ showTitle: true
+Even so, it’s right to question whether this actually does any good for the world. Are we really in need of better software? And beyond those in the most competitive industries, will perfect customer data actually make a difference to an everyday person’s ability to build a successful business?
+
+No and not really. However, that misses the point, unless you understand the secret master plan alluded to above. The strategy of PostHog is to get early stage software teams to use every single customer related product inside our platform, where we can easily generate a perfect customer data record. This means we can learn to configure and automate how all of these tools work. We can simplify the experience of running a business.
+
+Then with each successive product, we can reach a wider and wider market and further simplify the experience of running a business until anyone can do it.
+
+I’d like to address a repeated argument against shipping multiple products, let alone all of them.
+
+### You can build more than a product or two
+
+The world of tech is changing fast.
+
+First, there’s just more money at better valuations than when this advice was created. Software companies, us included, have more leverage than ever before - with capital deployed well, we can do more than our predecessors.
+
+Second, software development is far quicker.
+
+Third, we’ve managed this very well so far. 
 
 ## Our strategy
 
-### 1. Provide every tool needed to build successful products
+### 1. Be the source of truth for customer and product data
+
+Building a successful product is hard; doing so when you don't understand your customers is even harder. It's wild that no one has already provided a complete customer record. This has happened because the entire industry has focused on integration instead of consolidation.
+
+Traditionally, as companies scale, their data warehouse becomes the source of truth, and non-warehouse native tools (like product analytics) become less relevant as people lose trust in the data in them. Every company winds up with a huge mess of data spaghetti, with their business logic still spread across dozens or hundreds of tools.
 
-Building a successful product is hard; doing so when you don't understand your customers is even harder.
+We provide customer infrastructure - by providing _every_ customer data related tool in one place, we can:
+
+- Enhance the utility of all the tools when used together
+- Increase trust in data by eliminating complex data stacks
+- Automate everything better than anyone else can, by using AI across this wider data context
+- Continue to provide all the tools companies need as they grow
+
+### 2. Provide every tool needed to build successful products
 
 We aim to offer every tool software teams need to find, sell to, get users, understand who they are, support them and improve their product experience.
 
-### 2. Get in first
+By doing this, we wind up with the perfect customer data record. No integrations needed.
```

Comment:
Then why do we offer a CDP? should we explain that here?

Reply:
i think this is a bit edge casey (it's a necessary evil for now as we'll always have some gaps in tool coverage ie ad stuff) and too much depth for this page